tests/unit/modules/dcnm/fixtures/dcnm_intf_subint_configs.json

```diff
@@ -1,4 +1,15 @@
 {
+    "mock_fab_inv_data": {
+      "10.122.84.108": {
```

Comment:
Use private ip

Reply:
Ok. Will modify the code to use private IP addresses